src/pages/AddItem.js

```diff
@@ -4,6 +4,7 @@ import { useCollection } from 'react-firebase-hooks/firestore';
 import Swal from 'sweetalert2';
 import { useHistory } from 'react-router-dom';
 import Button from '../components/Button';
+import { isPatternLike, genericTypeAnnotation } from '@babel/types';
```

Comment:
Hi @jssckbl! Great job tackling the Sweet Alerts, it looks very nice with our pretty color palette ❤️  I was just wondering what this import statement is? Do we need it for this PR update?

Reply:
If I had to guess, and this is only guessing, that accidentally got thrown in by Jessica's editor. Mine does that dumb stuff all the time. You're typing something in and boom! all the sudden a random import at the top! I'll pull down and test to make sure everything works without it too.

Reply:
I was suspecting what James said is what happened. Thank you for offering to check out the functionality without it! Let us know what the result is.